addons/point_of_sale/models/pos_config.py

```diff
@@ -732,3 +744,21 @@ def setup_invoice_journal(self, company):
                 pos_config.write({'invoice_journal_id': invoice_journal_id.id})
             else:
                 pos_config.write({'module_account': False})
+
+    def get_limited_products_loading(self):
+        self.env.cr.execute("""
+            with pm as (select product_id,max(write_date) date from stock_quant group by product_id)
```

Comment:
Can you format this sql query? There is a good format described here: https://github.com/odoo/upgrade/wiki/How-To#sql-formatting